tests/e2e/util/kubeUtils.go

```diff
@@ -201,3 +201,60 @@ func GetIngressPod(n string) (string, error) {
 	_, err := retry.Retry(retryFn)
 	return ingress, err
 }
+
+// GetPodsName gets names of all pods in specific namespace and return in a slice
+func GetPodsName(n string) (pods []string) {
+	res, err := Shell("kubectl -n %s get pods -o jsonpath='{.items..metadata.name}'", n)
```

Comment:
I usually use
```
	pods, err := util.Shell("kubectl get pods -n %s -l app=echosrv -o jsonpath={.items[*].metadata.name}", ns)
```
I'm curious what's the difference between `[*].` and `..` / does yours work too ?

Reply:
Ah, you are right! .. is going to go recursively find all ".metadata.name" descent.
Reference: https://kubernetes.io/docs/user-guide/jsonpath/